Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Python image to use.
```

Comment:
Consider using a slim variant of the base image, like `python:3.9.6-slim-bullseye`, for smaller image size without sacrificing package availability compared to alpine.